tests/libvmi/sidecar.go

```diff
@@ -0,0 +1,55 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2023 Red Hat, Inc.
```

Comment:
Off topic from the PR, but this just caught my eye. I wonder why we don't copyright files on behalf of the KubeVirt Maintainers.

Reply:
Good point. I always thought that people put the copyright in from whichever company they worked for, but now I think that doesn't make sense.

However, it's been [forever like this](https://github.com/kubevirt/kubevirt/commit/96ca6ecbed3f64b01ed575b8ac4294bb0ca11f12). 

I think we should change it everywhere to [something similar the Kubernetes community uses](https://github.com/kubernetes/kubernetes/blob/master/cmd/clicheck/check_cli_conventions.go#L2C1-L2C38):

```suggestion
 * Copyright 2023 the KubeVirt Authors.
```

but IANAL and maybe we should firstly define what "the KubeVirt Authors" is?

@aburdenthehand WDYT?

Reply:
I've done a bit of research and suggestion by other LF communities is to use something like "Copyright the XYZ Authors". I've put it on next week's community meeting agenda so we can discuss this there.

Reply:
However, that's not a thing we want to solve in this PR.

Reply:
See https://github.com/kubevirt/kubevirt/pull/10645